packaging/datadog-agent/source/install_agent.sh

```diff
@@ -81,6 +81,8 @@ elif [ -f /etc/system-release -o "$DISTRIBUTION" == "Amazon" ]; then
 # Arista is based off of Fedora14/18 but do not have /etc/redhat-release
 elif [ -f /etc/Eos-release -o "$DISTRIBUTION" == "Arista" ]; then
     OS="RedHat"
+elif [ -f /etc/issue -o "$DISTRIBUTION" == "SUSE" ]; then
```

Comment:
I had to check, because I thought I had a reason for not choosing this. `/etc/SuSE-release` is deprecated:

```
greg@linux-ely7:/etc> cat SuSE-release 
SUSE Linux Enterprise Server 12 (x86_64)
VERSION = 12
PATCHLEVEL = 1
# This file is deprecated and will be removed in a future service pack or release.
# Please check /etc/os-release for details about this release.
```

We can use /etc/os-release or /etc/issue.